android/src/main/java/com/theoplayer/ReactTHEOplayerView.kt

```diff
@@ -444,4 +446,21 @@ class ReactTHEOplayerView(private val reactContext: ThemedReactContext) :
       eventEmitter.onFullscreenDidDismiss()
     }
   }
+
+  fun setAspectRatio(aspectRatio: String?) {
+    if (aspectRatio == this.aspectRatio) {
+      return
+    }
+    this.aspectRatio = aspectRatio
+    val value = getAspectRatioFromString(aspectRatio)
+    playerView?.player?.setAspectRatio(value)
+  }
```

Comment:
@kevin-kp you might want to get rid of backing field here
```suggestion
  var aspectRatio: String? 
    set(value) {
      if (field == value) {
        return
      }
      field = value
      getAspectRatioFromString(field).let {
        playerView?.player?.setAspectRatio(it)
      }
    }
```